### PR TITLE
uploading /MI-10-MEAN/Pratiksha/PratikshaA5/A5.html/A5.css/Sqpurple.gif

### PR DESCRIPTION
**Pull request by pratikshaghule**

### CODE REVIEW

File: Pratiksha/html/PratikshaA5/A5.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    <div class="whole ">  
+        <div >
+            <h1>
+                Pratiksha P Ghule
+            </h1>
+        </div>
+        <div class="bg">
+            <div class="about"> About me</div>
+            <p class="orange">
+                Pratiksha P Ghule is a director of brand marketing, with experience managing global teams and multi-million-dollar campaigns. Her background in brand strategy, visual design, and account management inform her mindful but competitive approach. Madison is fueled by her passion for understanding the nuances of cross-cultural advertising. She considers herself a ‘forever student,’ eager to both build on her academic foundations in psychology and sociology and stay in tune with the latest digital marketing strategies through continued coursework.</p>
```
Comment (VIP5614) [inline Pratiksha/html/PratikshaA5/A5.html]: p tag is not propar closed last line in first p tag

File: Pratiksha/html/PratikshaA5/A5.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+                <p>Her hunger for knowledge and determination to turn information into action has contributed to her most recent success at Rockwell Group, where she led international, award-winning campaigns for heavy-hitting brands, such as Puma, Gucci, and Rolex. Meanwhile, she vastly improved the productivity of her team by implementing strategic project management methods and ensuring a work-life balance for her department.
+                
+                Madison believes mindfulness in the workplace is key to success - a tenet she lives out through her interests in yoga, meditation, gardening, and painting. Madison is currently working as a freelance marketing director and is always interested in a challenge. Reach out to madisonblackstone@gmail.com to connect!</p>
+       
+        <div class="rt">  More about me,</div>
+            <div class="ul">
```
Comment (VIP5614) [inline Pratiksha/html/PratikshaA5/A5.html]: defined diffrent class name not defined ul class name

File: Pratiksha/html/PratikshaA5/A5.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                    <li><a href="/home/pratiksha/Downloads/Images/" target="_blank">Images</a></li>
+                </ul>
+            </div>
+            
+        
+        </div>
+    </div>
+        
+    </div>
+</body>
```
Comment (VIP5614) [inline Pratiksha/html/PratikshaA5/A5.html]: create proper stretcher html

**Review by sanketmod — APPROVED** [review]

File: Pratiksha/html/PratikshaA5/A5.html (hunk 1 above)
Comment (pratikshaghule) [reply Pratiksha/html/PratikshaA5/A5.html]: Done

File: Pratiksha/html/PratikshaA5/A5.html (hunk 3 above)
Comment (VIP5614) [reply Pratiksha/html/PratikshaA5/A5.html:51]: <html>
    <body>
     </body>
</html>
 aaye se hota hai Pratiksha ji
na ki aaye se
<html>
<body>
</body>
</html>
ok

Comment (VIP5614) [reply Pratiksha/html/PratikshaA5/A5.html]: please check

Comment (pratikshaghule) [reply Pratiksha/html/PratikshaA5/A5.html]: now check

File: Pratiksha/html/PratikshaA5/A5.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+       
+        <div class="rt">  More about me,</div>
+            <div class="ul">
+                <ul type ="square" >
+                    <li><a href="/home/pratiksha/Desktop/resume/Resume.html" target="_blank">Resume</a></li>
+                    <li><a href="/home/pratiksha/Desktop/hobbies/hobbies.txt" target="_blank">Hobbies</a></li>
+                    <li><a href="/home/pratiksha/Downloads/Images/" target="_blank">Images</a></li>
+                </ul>
+            </div>
+
```
Comment (vishwaraj1706) [inline Pratiksha/html/PratikshaA5/A5.html]: remove extra lines

**Review by mayank30041995 — APPROVED** [review]

File: Pratiksha/html/PratikshaA5/A5.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
+.bg{
+    background-image: url(https://tse3.mm.bing.net/th?id=OIP.mUc57NHscnXINcXYOp--WQHaEK&pid=Api&P=0&w=292&h=165);
```
Comment (MKadeer17) [inline Pratiksha/html/PratikshaA5/A5.css:2]: add link into double quotes.
like this: url("")

File: Pratiksha/html/PratikshaA5/A5.css [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.whole{
+
```
Comment (MKadeer17) [inline Pratiksha/html/PratikshaA5/A5.css:6]: remove this extra lines.

File: Pratiksha/html/PratikshaA5/A5.css [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    font-style: italic;
+    font-family:Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+    font-size: x-large;
+}
+.orange{
+    width:92%;
+    background-color: rgb(209, 109, 21);
+    box-shadow: 1px 2px black;
+    margin-left: -2px;
+
```
Comment (MKadeer17) [inline Pratiksha/html/PratikshaA5/A5.css:22]: this line also remove

File: Pratiksha/html/PratikshaA5/A5.html [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+            
+                <div class="rt">  More about me,</div>
+                    <div class="list">
+                        <ul type ="square" >
+                            <li><a href="/home/pratiksha/Desktop/resume/Resume.html" target="_blank">Resume</a></li>
+                            <li><a href="/home/pratiksha/Desktop/hobbies/hobbies.txt" target="_blank">Hobbies</a></li>
+                            <li><a href="/home/pratiksha/Downloads/Images/" target="_blank">Images</a></li>
+                        </ul>
+                    </div>
+
```
Comment (MKadeer17) [inline Pratiksha/html/PratikshaA5/A5.html]: remove this extra lines

File: Pratiksha/html/PratikshaA5/A5.html [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+                <div class="rt">  More about me,</div>
+                    <div class="list">
+                        <ul type ="square" >
+                            <li><a href="/home/pratiksha/Desktop/resume/Resume.html" target="_blank">Resume</a></li>
+                            <li><a href="/home/pratiksha/Desktop/hobbies/hobbies.txt" target="_blank">Hobbies</a></li>
+                            <li><a href="/home/pratiksha/Downloads/Images/" target="_blank">Images</a></li>
+                        </ul>
+                    </div>
+                    
+
```
Comment (MKadeer17) [inline Pratiksha/html/PratikshaA5/A5.html]: this line also

**Review by MKadeer17 — COMMENTED** [review]
Checkout only simple changes

**Review by vishwaraj1706 — APPROVED** [review]

File: Pratiksha/html/PratikshaA5/A5.css (hunk 7 above)
Comment (pratikshaghule) [reply Pratiksha/html/PratikshaA5/A5.css:22]: I want to overlap that paragraph thats why I had put margin

**Pull request merged**